frontend/app/src/pages/proposed-changes/items.tsx

```diff
@@ -0,0 +1,5 @@
+import { ProposedChangesPage } from "@/screens/proposed-changes/items";
+
+export function Component() {
+  return <ProposedChangesPage />;
```

Comment:
Ideally I think we should split the load and display logic on every page. Instead of having everything in the same component, we put all the fetch/permission and other related on /pages.
Then we pass only the data required by components under /screens who mainly just handle the display. WYDT?